packages/experiments/src/components/Chiclet/BaseChiclet.base.tsx

```diff
@@ -0,0 +1,41 @@
+import * as React from 'react';
+import {
+  BaseComponent,
+  customizable,
+  classNamesFunction
+} from '../../Utilities';
+import { Chiclet } from './Chiclet';
+import { ChicletSize } from './Chiclet.types';
+import { extractMetaTags } from './OpenGraph';
+import { IBaseChicletProps, IBaseChicletStyles, IBaseChicletStyleProps } from './BaseChiclet.types';
+
+const getClassNames = classNamesFunction<IBaseChicletStyleProps, IBaseChicletStyles>();
+
+@customizable('BaseChicletBase', ['theme'])
+export class BaseChicletBase extends BaseComponent<IBaseChicletProps, any> {
```

Comment:
>BaseChicletBase [](start = 13, length = 15)

BaseChicletBase is a somewhat odd naming pattern.  Is this common for OUIFR?

Reply:
Convention is that the base implementation (without styling) should only have the `.base.` extension on it and not have `Base` in the naming. (Exception for a few components that have not yet been converted over to Javascript styling.)

I agree that `BaseChiclet.base.tsx` is odd and I feel this should be named `Chiclet.base.tsx`. But I am also confused by the presence of `Chiclet.base.tsx` elsewhere in this PR and do not know what the difference is between that file and this one.

What I'd like to recommend is renaming all of these files to `Chiclet.*.tsx`, consistent with other components. If there is a specialized version of Chiclet, it can have its naming changed accordingly.